### PR TITLE
Update 06-AmsaleJonah.md

### PR DESCRIPTION
**Pull request by AmsaleJonah**
Release 11 planning

@githubteacher @brianamarie

### CODE REVIEW

**Comment by AmsaleJonah** [comment]
Testing

**Comment by AmsaleJonah** [comment]
@paulsuttle @AliFellows

**Comment by AmsaleJonah** [comment]
Can we plan this release over the coming weekend?  @paulsuttle

**Comment by ClaireS-001** [comment]
Test comment - cs  @AmsaleJonah

File: _slides/06-AmsaleJonah.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
----
-
-Caption here
-
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Release 11
```
Comment (tulikapgit) [inline _slides/06-AmsaleJonah.md:6]: nice change 🎖️

**Review by tulikapgit — APPROVED** [review]

**Pull request merged**